### PR TITLE
Add SSSD user_attributes configuration description

### PR DESCRIPTION
**Pull request by jvlcek**
https://bugzilla.redhat.com/show_bug.cgi?id=1380872

### CODE REVIEW

**Comment by jvlcek** [comment]
@abellotti Please review

**Comment by abellotti** [comment]
This LGTM !!!  Thanks Joe.

**Comment by abellotti** [comment]
@mfeifer for you reviewing/merging pleasures.  Thanks.

/cc @Fryguy

File: Rakefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 desc "Run tests by executing the ascii_binder build"
 task :test do
-  exec("bundle exec ascii_binder build --distro manageiq")
+  exec("bundle exec asciibinder build --distro manageiq")
```
Comment (Fryguy) [inline Rakefile]: LOL the easiest of all fixes, and I didn't do it 😳

Comment (jvlcek) [inline Rakefile]: I got your back @Fryguy

**Pull request merged**